backend/kesaseteli/applications/tests/test_excel_export.py

```diff
@@ -0,0 +1,75 @@
+import pytest
+from django.shortcuts import reverse
+
+from applications.enums import ApplicationStatus
+from applications.exporters.excel_exporter import export_applications_as_xlsx_output
+from applications.models import SummerVoucher
+
+
+def excel_download_url():
+    return reverse("excel-download")
+
+
+@pytest.mark.django_db
+def test_excel_view_get_with_authenticated_user(client, staff_user):
+    client.force_login(staff_user)
+    response = client.get(excel_download_url())
+    assert response.status_code == 200
+
+
+@pytest.mark.django_db
+def test_excel_view_get_with_unauthenticated_user(client, user):
+    client.force_login(user)
+    response = client.get(excel_download_url())
+    assert response.status_code == 302
+
+
+@pytest.mark.django_db
+def test_excel_view_download_unhandled(client, staff_user, summer_voucher):
+    client.force_login(staff_user)
```

Comment:
I'd create a `staff_client` and  a `user_client` fixture to avoid repeating this line